docs/programming-guide.md

```diff
@@ -1345,14 +1345,17 @@ therefore be efficiently supported in parallel. They can be used to implement co
 MapReduce) or sums. Spark natively supports accumulators of numeric types, and programmers
 can add support for new types.
 
-If accumulators are created with a name, they will be
-displayed in Spark's UI. This can be useful for understanding the progress of
-running stages (NOTE: this is not yet supported in Python).
+As a user, you can create `Accumulators` that are both named and unnamed. Named accumulators will display in Spark's UI along with their running totals during execution. As seen in the image below, a named accumulator (in this instance `counter`) will display
```

Comment:
Actually, one nit: "Accumulators" isn't a class name here and shouldn't be back-ticked. Nit 2: say "either named or unnamed", or else just say "that can be named."

Also I might not describe accumulators as keeping a "running total". Maybe just say "Spark displays the value for each accumulator used by a task in the "Tasks" table ..."